auparse/auparse.c

```diff
@@ -398,6 +400,32 @@ void print_lol(char *label, au_lol *lol)
 
 
 /* General functions that affect operation of the library */
+
+/*
+ * au_setup_userspace_configitems - load userspace configuration items from auditd.conf
+ *
+ * Args:
+ *  au - pointer to auparseing state structure
+ *
+ * Rtns:
+ *      void
+ */
+static void au_setup_userspace_configitems(auparse_state_t *au)
+{
+	struct daemon_conf config;
+
+        /* Load config so we know where logs are */
+	set_aumessage_mode(au, MSG_STDERR, DBG_NO);
+	aup_load_config(au, &config, TEST_SEARCH);
+
+	// Now over-ride the interal user space configuration items
+
+	if (config.end_of_event_timeout != EOE_TIMEOUT)
+		eoe_timeout = (time_t)config.end_of_event_timeout;
+
```

Comment:
I'm wondering if config.log_file ought to be kept so that it can be given to setup_log_file_array() if needed? Also, the if really isn't necessary since if they are the same it's harmless.

Reply:
I thought about only having one struct daemon_conf structure the same way that ausearch/aureport does, but decided on minimal change ... that would have been a bit more change given we only call setup_log_file_array for only one of a number of ausource_t types.

And yes, we can ditch the if()

Reply:
The if has been removed